packages/repository/test/unit/decorator/model-and-relation.ts

```diff
@@ -116,6 +127,7 @@ describe('model decorator', () => {
     });
     expect(meta.id).to.eql({type: 'string', id: true, generated: true});
     expect(meta.isShipped).to.eql({type: Boolean});
+    expect(meta.items).to.eql({type: Product, array: true});
```

Comment:
Can you please explain why here `type` is `Product` which is a Class, but https://github.com/strongloop/loopback-next/pull/806/files#diff-aa9599539cae81d1bbc3cdb480ecdb09R81 we define it as a function?

Reply:
All objects are Functions, and this is the only way we can properly handle class constructors in a way that doesn't resort to the use of `any`. If you look at most of our other decorators, we use the same approach.

Reply:
Thanks 👍